### PR TITLE
add vertical timeline view

### PR DESCRIPTION
**Pull request by denizsubasi**
#48 
Added timeline vertical feature.
Changed readme timeline-view image.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
Timeline vertical support

## How Has This Been Tested?
test on android emulator.

## Screenshots (if appropriate):

timeline-view

### CODE REVIEW

File: libraries/timeline-view/src/main/java/com/trendyol/timelineview/TimelineItemViewState.kt [hunk 1]
```diff
@@ -22,15 +21,15 @@ class TimelineItemViewState(
 
     fun getTextColor(): Int = Color.parseColor(timelineItem.textColor)
 
-    fun getLeftLineColor(): Int = Color.parseColor(timelineItem.leftLineColor ?: hexColorWhite)
+    fun getLeftLineColor(): Int = Color.parseColor(timelineItem.startLineColor ?: hexColorWhite)
```
Comment (bsobe) [inline libraries/timeline-view/src/main/java/com/trendyol/timelineview/TimelineItemViewState.kt:24]: Your function not responsible for only left/right. Please find more declarative function naming.

**Review by bsobe — CHANGES_REQUESTED** [review]
Thanks for contributing.

File: libraries/timeline-view/src/main/java/com/trendyol/timelineview/TimelineItemViewState.kt (hunk 1 above)
Comment (denizsubasi) [reply libraries/timeline-view/src/main/java/com/trendyol/timelineview/TimelineItemViewState.kt]: wrong function names fixed

**Review by bsobe — APPROVED** [review]
Thanks for renaming.

**Review by furkant9 — APPROVED** [review]

**Review by selimtoksal — APPROVED** [review]

**Pull request merged**